docs/how-to/file-uploads.md

```diff
@@ -9,7 +9,7 @@ title: File Uploads
 <br/>
 <br/>
 
-Handle file uploads in your React Router applications. This guide uses some packages from the [Remix The Web][remix-the-web] project to make file uploads easier.
+Handle file uploads in your React Router applications. This guide uses some packages from the [Remix 3][remix-3] project to make file uploads easier.
```

Comment:
Just FYI, [I did already update these 2 docs](https://github.com/remix-run/react-router/commit/1752f4f9e7b30ac5e8de55497ebeb62debf96da2) on `main`, which is why they're not reflected here in `dev`

Probably best to cherrypick those changes into these files, or just forgo those changes